mypy/build.py

```diff
@@ -989,13 +1064,6 @@ def validate_meta(meta: Optional[CacheMeta], id: str, path: Optional[str],
     if data_mtime != meta.data_mtime:
         manager.log('Metadata abandoned for {}: data cache is modified'.format(id))
         return None
-    deps_mtime = None
-    if manager.options.cache_fine_grained:
```

Comment:
Why was this removed?

Reply:
Because we can't write out the deps file before the meta file anymore, so it can't have the mtime. We store mtime in another cache file.